navext/navext-runtime/src/main/java/com/foursoft/harness/navext/runtime/io/validation/LogValidator.java

```diff
@@ -96,21 +97,7 @@ public boolean validate(final Source source) {
     /**
      * ErrorLocation holds the line and message of the found error
      */
-    public static class ErrorLocation {
-        public final int line;
-        public final String message;
-
-        public ErrorLocation(final int line, final String message) {
-            this.line = line;
-            this.message = message;
-        }
-
-        @Override
-        public String toString() {
-            return "ErrorLocation{" +
-                    "line=" + line +
-                    ", message='" + message + '\'' +
-                    '}';
-        }
+    public record ErrorLocation(int line, String message) {
     }
```

Comment:
@RainerGanss: I converted this static inner class to a record. This itself would be a breaking change since `ErrorLocation#line` and `ErrorLocation#message` are not compiling anymore.

Should we keep this and if so, how do we treat it regarding the upcomming release?

Reply:
no please revert this. We won`t make a minor release just for this.